### ISSUE TITLE
The training set of the Libriphrase

### ISSUE THREAD

**Issue by AIKing111** [ncsoft/PhonMatchNet#5]
Hello, I would like to know how you obtained the training set of the Libriphrase dataset, as it seems that the official source has only released the recipe for the test set.

**Comment by LeeYongHyeok** [comment]
Hello @AIKing111, We followed the official LibriPhrase repository to create csv files and then loaded them into our dataset/libriphrase.py code.
